dev/dags/example_map_index_template.yml

```diff
@@ -1,4 +1,4 @@
-# Requires Airflow 2.7 or higher
+# Requires Airflow 2.9 or higher
```

Comment:
I checked that the `map_index_template` field was introduced in PR https://github.com/apache/airflow/pull/36797 which was released only in Airflow 2.9, hence, updating the same here.